fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1554,6 +1554,9 @@ protected void runAfterCatalogReady() {
     public void createReplayer() {
         replayer = new Daemon("replayer", REPLAY_INTERVAL_MS) {
             private JournalCursor cursor = null;
+            // avoid numerous 'meta out of date' log
+            private long lastMetaOutOfDateLogTime = 0;
```

Comment:
It seems there is no place to update this variable

Reply:
Good point. I have just removed this member.